internal/nodes/avi_model_nodes.go

```diff
@@ -875,11 +894,21 @@ func (v *AviHttpPolicySetNode) CalculateCheckSum() {
 	if v.HeaderReWrite != nil {
 		checksum = checksum + utils.Hash(utils.Stringify(v.HeaderReWrite))
 	}
+
 	if lib.GetGRBACSupport() {
 		checksum += lib.GetMarkersChecksum(v.AviMarkers)
 	}
+
 	v.CloudConfigCksum = checksum
 }
+func (v *AviHostPathPortPoolPG) CalculateCheckSum() {
```

Comment:
Is this being used ?

Reply:
Yes.  it is used in l7_translator as well as in evh_node file to compute checksum associated with hpp map. Sample call is: httpPGPath.CalculateCheckSum()